docs/guided-tour/blueprints/external-blueprint/README.md

```diff
@@ -23,34 +23,55 @@ registry, from where the Landscaper can access it. You can find the commands, wh
 in this script: [commands/push-blueprint.sh](./commands/push-blueprint.sh).
 
 
-## Components and Component Descriptors
+## Components
 
 > **_NOTE:_** **To follow along the following section, be sure to set the `useOCM: true` feature switch in the 
 > values.yaml, as shown [here](https://github.com/gardener/landscaper/blob/master/docs/installation/install-landscaper-controller.md#configuration-through-valuesyaml).**
 
-An Installation may reference its blueprints via so-called
-[component-descriptors](../../../concepts/Glossary.md#_component-descriptor_).  A component descriptor describes a
-component, or rather, a specific component version. In general, a component version is a container for all required
-resources for the deployment of a specific version of an application or software system. In this example, the
-application is the hello-world application deployable with the landscaper through an external blueprint. Thereby, the
-external blueprint is the only resource required for the deployment and thus, the only resource contained in the
-component version. A component version may either contain a resource through referencing it at an external location 
-(such as an OCI registry) or through embedding it as a local blob. A quick introduction to this concepts can be found at
-the end of this section. 
+To consume the blueprint from such a referencable location, it has to be contained in a component.
 
```

Comment:
I found these explanations very good and helpful. You introduced them some time before and I was very happy about them.